### ISSUE TITLE
Segment with "email open" filter

### ISSUE THREAD

**Issue by PamMarciano** [mautic/mautic#7546]
Mautic 2.15.1
PHP 7.2.18
Browser Safari 12.1.1

I created a new segment with filter on "email open".

The segment always shows 0 contacts (should have a lot!).

I get the following error messages when the cron job update the segments:

[Doctrine\DBAL\Exception\InvalidFieldNameException]
An exception occurred while executing 'SELECT count(leadIdPrimary) count, max(leadIdPrimary) maxId, min(leadIdPrimary) minId F
ROM (SELECT DISTINCT l.id as leadIdPrimary, HSnzyufd.lead_id AS HSnzyufd_lead_id FROM maleads l LEFT JOIN malead_lists_leads H
Snzyufd ON HSnzyufd.lead_id = l.id and (HSnzyufd.leadlist_id = 84) WHERE (EXISTS(SELECT NULL FROM maemail_stats jygWzqCD WHERE
(jygWzqCD.lead_id = l.id) AND (email_stats.is_read = 1) AND (jygWzqCD.email_id IN (?, ?, ?, ?)))) AND (HSnzyufd.lead_id IS NU
LL)) sss' with params [65, 69, 56, 52]:

SQLSTATE[42S22]: Column not found: 1054 Unknown column 'email_stats.is_read' in 'where clause'

[Doctrine\DBAL\Driver\PDOException]
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'email_stats.is_read' in 'where clause'

[PDOException]
SQLSTATE[42S22]: Column not found: 1054 Unknown column 'email_stats.is_read' in 'where clause'

I updated the database schema but the issue remains.

Otherwise evertything works fine with this Mautic installation.

**Comment by PamMarciano** [comment]
Mautic 2.15.1
PHP 7.2.18
Browser Safari 12.1.1

I created a new segment with filter on "email open".

However the segment never updates and I get the following error messages when the cron job update the segments:

[Doctrine\DBAL\Exception\InvalidFieldNameException]
  An exception occurred while executing 'SELECT count(leadIdPrimary) count, max(leadIdPrimary) maxId, min(leadIdPrimary) minId F
  ROM (SELECT DISTINCT l.id as leadIdPrimary, HSnzyufd.lead_id AS HSnzyufd_lead_id FROM maleads l LEFT JOIN malead_lists_leads H
  Snzyufd ON HSnzyufd.lead_id = l.id and (HSnzyufd.leadlist_id = 84) WHERE (EXISTS(SELECT NULL FROM maemail_stats jygWzqCD WHERE
   (jygWzqCD.lead_id = l.id) AND (email_stats.is_read = 1) AND (jygWzqCD.email_id IN (?, ?, ?, ?)))) AND (HSnzyufd.lead_id IS NU
  LL)) sss' with params [65, 69, 56, 52]:

  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'email_stats.is_read' in 'where clause'

  [Doctrine\DBAL\Driver\PDOException]
  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'email_stats.is_read' in 'where clause'

 [PDOException]
  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'email_stats.is_read' in 'where clause'

I updated the database schema but the issue remains.

Otherwise evertything works fine with this Mautic installation.
